src/main/java/seedu/address/model/person/GenderContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code Email} matches any of the keywords given.
```

Comment:
I think you forgot to edit the javadoc to `gender`

Reply:
right thnks!!